### ISSUE TITLE
c8000v fails to start - launch.py: error: unrecognized arguments: --hostname node1 --connection-mode tc

### ISSUE THREAD

**Issue by gdluca** [srl-labs/containerlab#2095]
I built a docker image for the Cisco c8000v 17.13.01a following the vrnetlab procedure. If I create a new container directly from docker, router boots fine.

```
docker run -d --privileged --name my-c8000v-router vrnetlab/vr-c8000v
```

I am using the following config

```
name: c8000v
topology:
  nodes:
    node1:
      kind: cisco_c8000v
      image: vrnetlab/vr-c8000v:17.13.01a
    node2:
      kind: cisco_c8000v
      image: vrnetlab/vr-c8000v:17.13.01a

  links:
    - endpoints: ["node1:eth1", "node2:eth1"]
```
  
Checking the containers logs, the two nodes stop at the same point:

- node1
```
[pktdrop@zon3 c8000v]$ sudo docker logs clab-c8000v-node1
usage: launch.py
       [-h]
       [--trace]
       [--username USERNAME]
       [--password PASSWORD]
       [--install]
launch.py: error: unrecognized arguments: --hostname node1 --connection-mode tc
```
- node2
```
[pktdrop@zon3 c8000v]$ sudo docker logs clab-c8000v-node2
usage: launch.py
       [-h]
       [--trace]
       [--username USERNAME]
       [--password PASSWORD]
       [--install]
launch.py: error: unrecognized arguments: --hostname node2 --connection-mode tc
```
Manually checking that launch.py, I confirmed that there is not --hostname or --connection-mode

```
[pktdrop@zon3 docker]$ ./launch.py --help
usage: launch.py [-h] [--trace] [--username USERNAME] [--password PASSWORD] [--install]

optional arguments:
  -h, --help           show this help message and exit
  --trace              enable trace level logging
  --username USERNAME  Username
  --password PASSWORD  Password
  --install            Install C8000v
```

**Comment by hellt** [comment]
Hi
You have to use hellt/vrnetlab to build compatible images

**Comment by gdluca** [comment]
Hi!
that was quick, thanks :)
Yes, I used hellt/vrnetlab to build the docker image.
I don't know what I am doing here but I ran a quick test to see if clab would at least generate the two nodes commenting out these two lines. This time, clab creates two containers and both do not exit with above launch.py error. 

https://github.com/srl-labs/containerlab/blob/main/nodes/vr_c8000v/vr-c8000v.go#L70-L71

```
	//n.Cfg.Cmd = fmt.Sprintf("--username %s --password %s --hostname %s --connection-mode %s --trace",
	//	n.Cfg.Env["USERNAME"], n.Cfg.Env["PASSWORD"], n.Cfg.ShortName, n.Cfg.Env["CONNECTION_MODE"])
```

the username and password are the same as vrnetlab set when building the docker image from the qcow2([here](https://github.com/vrnetlab/vrnetlab/blob/master/c8000v/docker/launch.py#L204-L205)). however, I can SSH into one node but the other the SSH hangs or something.
